ingest/sqs/chain_info/usecase/chain_info_usecase.go

```diff
@@ -13,6 +16,10 @@ type chainInfoUseCase struct {
 	redisRepositoryManager mvc.TxManager
 }
 
+// The max number of seconds allowed for there to be no updates
+// TODO: epoch???
+const MaxAllowedHeightUpdateTimeDeltaSecs = 30
```

Comment:
We're ignoring health being down on epoch rn correct?

Reply:
Yes, I will work with Grant on defining the healthcheck behavior during epoch